library/src/rocblas_auxiliary.cpp

```diff
@@ -71,6 +74,10 @@ extern "C" rocblas_status rocblas_create_handle(rocblas_handle* handle)
     try
     {
         *handle = new _rocblas_handle();
+
+        static std::once_flag tensileIntiFlag; 
+        std::call_once(tensileIntiFlag, tensileInitialize);
+
```

Comment:
Instead of using `std::call_once()`,  a simpler and faster method is to do:
```
static int dummy = (tensileInitialize(), 0);
```
You do not need `std::call_once`, `#include <thread>`, or `#include <mutex>` anymore.

In C++11 this is guaranteed to be thread-safe.